src/uglify/serialization.js

```diff
@@ -0,0 +1,46 @@
+const toType = value => (Object.prototype.toString.call(value).slice(8, -1));
```

Comment:
Since `RegExp` or` Function` can not be serialized, they need special handling:

```js
worker.send({ extractComments: /foo/ });
```

```js
// worker
process.on('message', (message) => {
    console.log(message); // -> { extractComments: {} } 😢
})
```

Use serialization :

```js
worker.send(encode({ extractComments: /foo/ }));
```

```js
// worker
process.on('message', (message) => {
    message = decode(message);
    console.log(message); // -> { extractComments: /foo/ } 😊
})
```